app/assets/javascripts/admin/addon/adapters/theme.js

```diff
@@ -1,9 +1,10 @@
 import RestAdapter from "discourse/adapters/rest";
 
-export default RestAdapter.extend({
+export default class Theme extends RestAdapter {
+  jsonMode = true;
```

Comment:
Do you know why just some adapters use the `jsonMode`? What does it mean? Should this be in `discourse/adapters/rest`?

Reply:
The default for our RestAdaptor is to use FormData when sending requests to the server. Individual models can opt-in to JSON mode, which means they can send more complex data structures.

Ideally yeah, I think we'd be using JSON everywhere. But it's a potentially breaking change, so definitely beyond the scope of this PR.